src/cmd/vm.rs

```diff
@@ -118,11 +239,40 @@ pub struct ArgsPush {
     #[argh(option, short = 'a')]
     android_build: String,
 
-    /// extra arguments
+    /// extra arguments. You can pass other options like --extra-args "options".
     #[argh(option)]
-    extra_args: Vec<String>,
+    extra_args: Option<String>,
 }
 
-fn run_push(_args: &ArgsPush) -> Result<()> {
+fn run_push(args: &ArgsPush) -> Result<()> {
+    let mut options = vec!["android_build", &args.android_build];
+    let mut extra_args = convert_str_to_vec(&args.extra_args);
+    options.append(&mut extra_args);
+
+    run_betty("push", &options)
+}
+
+fn convert_str_to_vec(input: &Option<String>) -> Vec<&str> {
+    match input {
+        Some(i) => i.split_whitespace().collect(),
+        None => Vec::new(),
+    }
+}
+
+fn run_betty(subcommand: &str, options: &Vec<&str>) -> Result<()> {
+    let cmd = Command::new("./betty.sh")
```

Comment:
It is possible that a user is not just under the directory which has betty.sh. Could you implement a logic to determine the full path of betty from the current directory and make this command work anywhere in the checkout?

Reply:
Sure, I added the `--repo` option that indicates the working directory for the child process and updated it to work anywhere!  
https://github.com/google/lium/pull/147/commits/20757709ad69453465f8719cefe036cf3ecf8f3b